app/views/targets/_facebook_logout.haml

```diff
@@ -0,0 +1,25 @@
+:javascript
+  window.fbAsyncInit = function() {
+    FB.init({
+      appId      : '1653169284778740',
```

Comment:
move this id to an ENV variable
something like `ENV.fetch('FACEBOOK_APP_ID')`
then you can have the application.yml define this for each environment like staging or localhost